pkgs/by-name/th/theforceengine/package.nix

```
@@ -14,20 +15,14 @@
   zenity,
   withEditor ? true,
 }:
-let
-  # package depends on SDL2main static library
-  SDL2' = SDL2_classic.override {
-    withStatic = true;
-  };
-in
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "theforceengine";
   version = "1.22.300";
 
   src = fetchFromGitHub {
     owner = "luciusDXL";
     repo = "TheForceEngine";
-    rev = "v${version}";
+    rev = "v${finalAttrs.version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```
Since you're modernising the package already, might as well switch to `tag`.